<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewRecordDialog</class>
 <widget class="QDialog" name="AddNewRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>  添加新词条</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix1/rc/actions/add.png</normaloff>:/new/prefix1/rc/actions/add.png</iconset>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/dialog_ok_apply.png</normaloff>:/new/prefix1/rc/actions/dialog_ok_apply.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="InfoGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>添加一条新的词条</string>
   </property>
   <widget class="QLabel" name="word">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>单词：</string>
    </property>
   </widget>
   <widget class="QLabel" name="mean">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>词义：</string>
    </property>
   </widget>
   <widget class="QLabel" name="other">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>相关信息：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WordLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="MeanTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="OtherTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lektion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>16</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>单元：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LektionLineEdit">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/editdelete.png</normaloff>:/new/prefix1/rc/actions/editdelete.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
